### PR TITLE
Fixed assets to use proper images

### PR DESCRIPTION
**Pull request by jklypchak13**

### CODE REVIEW

**Comment by jklypchak13** [comment]
I also added an array for multiple ground objects (you're welcome Nick H.)

File: src/game/levels/__init__.py [hunk 1]
```diff
@@ -12,12 +12,14 @@
...
 # Intialize Useful Constants
 PATH_TO_LEVELS: str = pathlib.Path(__file__).parent.absolute()
-PLATFORM_IMAGE_URL: str = Path(
+ASSET_PATH: str = Path(
     __file__
-).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep + "GRASS_TILE.png"
-ENEMY_IMAGE_URL: str = Path(
-    __file__
-).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep + "greek_soldier_walk1.png"
+).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep
```
Comment (NickHackman) [inline src/game/levels/__init__.py]: Appreciate you tidying up the `asset path` here, it was a major eye sore.

File: src/game/levels/__init__.py [hunk 2]
```diff
@@ -12,12 +12,14 @@
...
+PLATFORM_IMAGE_URL: str = ASSET_PATH + "wood_platform.png"
+HAY_IMAGE_URL: str = ASSET_PATH + "hay_bale.png"
+
+GROUND_IMAGE_URL: str = ASSET_PATH + "GRASS_TILE.png"
+ENEMY_IMAGE_URL: str = ASSET_PATH + "greek_soldier_walk1.png"
```
Comment (NickHackman) [inline src/game/levels/__init__.py]: I prefer string interpolation over string concat, but I doubt it makes any difference.

**Review by NickHackman — APPROVED** [review]

**Comment by NickHackman** [comment]
Merging...

**Pull request merged**